src/layouts/sidebar-sidecar/server.ts

```diff
@@ -51,6 +51,7 @@ export function getStaticGenerationFunctions<
   const loaderOptions: RemoteContentLoader['opts'] = {
     product: productSlugForLoader,
     basePath: basePathForLoader,
+    enabledVersionedDocs: true,
```

Comment:
🪄

Reply:
💭 Any reason to **not** use `process.env.ENABLE_VERSIONED_DOCS` or `isVersionedDocsEnabled`?

Reply:
Hmm, I don't see any reason _not_ to hardcode it at the moment. The intent behind that env var was mostly just to facilitate the rollout. So I guess it's tech debt now 🙃